website/components/demo-block.vue

```diff
@@ -241,8 +241,8 @@ ${this.codepen.style}
 
     scrollHandler() {
       const { top, bottom, left } = this.$refs.meta.getBoundingClientRect()
-      this.fixedControl = bottom > document.documentElement.clientHeight &&
-          top + 44 <= document.documentElement.clientHeight
+      this.fixedControl = bottom + 44 > document.documentElement.clientHeight &&
+          top <= document.documentElement.clientHeight
```

Comment:
Maybe make `44` as some constant like `expanderHeight` to make more readable.

Reply:
Okey. it will be changed later.